paper.md

```diff
@@ -0,0 +1,87 @@
+    index: 1
+date: 03 May 2021
+bibliography: paper.bib
+
+---
+
+# Summary
+
+`scikit-COSMO` (skcosmo) focuses on machine learning methods that are of particular use (but not limited
+                                                                                        to) the field of materials science and computational chemistry. Currently, many
+machine learning studies employ the foundational `scikit-learn` repository, a
+collection of widely applicable machine learning algorithms and methods.
+Written in the same style and guidelines as `scikit-learn`, we aim to provide
+users the ability to seamlessly include these methods in machine learning
+workflows alongside those from `scikit-learn`.
+
+# Statement of need
+
+In `scikit-COSMO`, we provide machine learning algorithms developed for a specific
+community: those who use machine learning to understand or represent collections
+of atoms or molecules. The emphasis put upon each subclass of machine learning
+algorithms varies within different subfields. For example, in applying machine
+learning to collections of atoms, there is a necessary focus on the
+representation of data in a numerical format for machine learning pipelines,
+known as _featurization_. Many machine learning studies represent samples using
+real-world measurements or tests, effectively limiting the number of features
+based upon the resources available. When representing a collection of atoms,
+there is both a variety of featurization techniques to employ and a wealth of
+tunable hyperparameters within each technique. Thus, the size of the vectors
+which represent atoms is almost unbounded, and it becomes necessary to assess
+their utility, remove redundancies, or condense them. Currently, `scikit-COSMO`
+contains algorithms to address these specific problems, all of which were
+previously-unavailable within open-source repositories.
+
+- Feature Reconstruction Measures[@Goscinski2021] - both global and local measures
+of the information density for a given representation. This set of
+unsupervised algorithms is of particular use in the choice of representation
+and hyperparameter optimization.
+- Feature and Sample Selection[@Imbalzano2018
+                               @Cersonsky2021] - methods focused on determining
+a diverse or information-rich subset of features or samples for machine
+learning problems.
+- Partially-Supervised Dimensionality Reduction[@deJong1992
+                                                @Helfrecht2020] - linear and
+non-linear techniques to combine features into latent-space projections
+(similar to PCA) that also incorporate target information. This is of particular
+use when condensing features prior to property regression or constructing
+structure-property maps, such as those analyzable via[@Fraux2020].
+
+Note: `scikit-COSMO` is not intended to compute atomic descriptors, as this is
+the role of the companion package `librascal` [@Musil2021] or similar, such as
+QUIP[@quip].
```

Comment:
```suggestion
`scikit-COSMO` itself does not compute atomic descriptors directly, instead taking as input matrices of descriptor computed by other software such as `librascal` [@Musil2021],
`QUIP`[@quip], `DScribe`[https://github.com/SINGROUP/dscribe] and similar software.
```

Reply:
```suggestion
Note: `scikit-COSMO` itself does not compute atomic descriptors directly, and instead takes as input descriptors compute by other software such as the companion package `librascal` [@Musil2021] or similar, including
QUIP[@quip] and `DScribe`[https://github.com/SINGROUP/dscribe].
```

We should highlight the increased compatibility that will *hopefully* be achieved with librascal.

Reply:
> We should highlight the increased compatibility that will hopefully be achieved with librascal.

I disagree, we should strive for good compatibility with more than just software we are developing ourself, because this makes skcosmo useful to more people.

Reply:
Fair point. How about this?
```suggestion
Note: `scikit-COSMO` itself does not compute atomic descriptors directly, and instead takes as input descriptors computed by other software such as `librascal` [@Musil2021],
QUIP[@quip], and `DScribe`[https://github.com/SINGROUP/dscribe].
```

Reply:
That sounds good to me 👍 . I don't like the `Note:` prefix, but if you want to keep I'll be fine!

Reply:
Bof, it's an idiosyncrasy that I picked up from @ceriottm, and a habit I'm trying to break :D